examples/grpc/grpc_example.cfg

```
@@ -0,0 +1,20 @@
+
+grpc_port: 9314
+
+probe {
+  name: "grpc_generic"
+  type: GRPC
+  targets {
+    # Hit another cloudprober instance running with grpc_server enabled.
+    host_names: "localhost:9314"
+  }
+
+  grpc_probe {
+    insecure_transport: true
+    method: GENERIC
+    request {
+      call_service_method: "cloudprober.Cloudprober.ListProbes"
+      body: "{}"
+    }
+    }
+}
```

Minor indentation fix:
```suggestion
    }
  }
}
```